draft-ietf-tls-dtls-connection-id.md

```diff
@@ -345,6 +345,33 @@ data the following modification is made to the additional data calculation.
                       length_of_DTLSInnerPlaintext;
 ~~~
 
+# Peer Address Update {#peer-address-update}
+
+When a datagram with a CID is received that has a source address different than the one currently associated with the DTLS connection, the receiver MUST NOT replace the source address currently stored in the association with the source address specified in the received datagram unless the following conditions are met:
+
+- The received datagram has been cryptographically verified using the DTLS record layer processing procedures.
```

Comment:
...and any duplicate detection (?)

Reply:
We list the different requirements separately.